src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,29 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing `Cloneable` and providing a custom `clone` method is generally not recommended for entities managed by Hibernate. This could lead to unexpected behavior, especially when dealing with lazy-loaded associations and proxies. If you need to create copies of entities, consider using copy constructors or dedicated 'copy' methods that make the copying explicit and clear.